allensdk/brain_observatory/ecephys/behavior_ecephys_session.py

```diff
@@ -359,7 +359,6 @@ def from_json(
         -------
         Instantiated `BehaviorEcephysSession`
         """
-        session_data = session_data['session_data']
```

Comment:
I think for better or worse, we need this line.
 
 If you want to change it, I think you need to change the call signature for `BehaviorEcephysSession.from_json` to take `dict_repr` instead of `session_data` and then replace this line with
 
 ```
 session_data = dict_repr['session_data']
 ```

Reply:
with this line we are getting an error:
```
 File "/local1/pyprojects/AllenSDK/allensdk/brain_observatory/nwb/nwb_utils.py", line 180, in _write_nwb
    session_data=session_data, **from_json_kwargs)
  File "/local1/pyprojects/AllenSDK/allensdk/brain_observatory/ecephys/behavior_ecephys_session.py", line 362, in from_json
    session_data = session_data['session_data']
KeyError: 'session_data'

```

Reply:
Okay. Then we need a fix to a unit test. I'll push it up.